util/gdal/src/gdal_util.cc

```diff
@@ -105,4 +105,11 @@ std::string ConvertToWkt(std::string_view shp_file_path) {
 
     return std::string(wkt_shp);
 }
+
+void AddMetadataTo(GDALDataset* ds, const common::metadata::Container& md) {
+    for (const auto& m : md.GetValues()) {
```

Comment:
Probably should use std::map, otherwise the order of metadata depends on the linked C++ version.

Reply:
Seems non issue, gdal sorts itself.